pkgs/development/idris-modules/lightyear.nix

```diff
@@ -0,0 +1,28 @@
+{ build-idris-package
+, fetchFromGitHub
+, prelude
+, base
+, effects
+, lib
+, idris
+}:
+
+build-idris-package {
+  name = "lightyear";
```

Comment:
To improve comparability of versions, they should be date of the commit as "yyyy-mm-dd" if they are not an official release.